easybuild/easyconfigs/c/CuPy/CuPy-11.4.0-foss-2021b-CUDA-11.4.1.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'PythonBundle'
+
+name = 'CuPy'
+version = '11.4.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://cupy.dev'
+description = "CuPy is an open-source array library accelerated with NVIDIA CUDA."
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('CUDA', '11.4.1', '', SYSTEM),
+    ('cuDNN', '8.2.2.26', versionsuffix, SYSTEM),
+    ('NCCL', '2.10.3', versionsuffix),
+    ('cuTENSOR', '1.6.1.5', versionsuffix, SYSTEM),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_LOWER_SOURCE]}
+
+exts_list = [
+    ('hypothesis', '6.54.6', {
+        'checksums': ['2d5e2d5ccd0efce4e0968a6164f4e4853f808e33f4d91490c975c98beec0c7c3'],
+    }),
+    ('fastrlock', '0.5', {
+        'checksums': ['9ae1a31f6e069b5f0f28ba63c594d0c952065de0a375f7b491d21ebaccc5166f'],
+    }),
+    ('cupy', version, {
+        'module': False,
+        'runtest': "cd tests && pytest",
+        'testinstall': True,
```

Comment:
testinstall is not picked up here due to being used in an extension. We need a fix in framework I guess

Reply:
https://github.com/easybuilders/easybuild-easyblocks/pull/2872 fixed here

Reply:
No, it doesn't because framework doen't progress testinstall up to pythonpackage.py when it's in an extension.

Reply:
meh, managed to pick the wrong version of that easyblock :-(